Validation/HGCalValidation/scripts/makeHGCalValidationPlots.py

```diff
@@ -48,14 +51,23 @@ def main(opts):
         val = SeparateValidation([sample], opts.outputDir[0])
     htmlReport = val.createHtmlReport(validationName=opts.html_validation_name[0])   
 
+    #layerClusters
     if opts.collection==layerClustersGeneralLabel:
 	hgclayclus = [hgcalPlots.hgcalLayerClustersPlotter]
 	hgcalPlots.append_hgcalLayerClustersPlots("hgcalLayerClusters", "Layer Clusters", extendedFlag)
         val.doPlots(hgclayclus, plotterDrawArgs=drawArgs)
+    #simClusters
+    elif (opts.collection == simClustersGeneralLabel) :
```

Comment:
Not necessary, but can we try to make coding in the same style, at least in the same file? For example, we have a mixing between
- opts.collection==layerClustersGeneralLabel:
- (opts.collection == simClustersGeneralLabel) :

Reply:
https://github.com/cms-sw/cmssw/commit/34d750f072c38954edf9eb0699ac505517583778